### PR TITLE
create model erfnet_resa

### PR DESCRIPTION
**Pull request by LittleJohnKhan**
ERFNetEncoder is copied from `utils/models/segmentation/erfnet.py`, then add pretrained-weights initialization

### CODE REVIEW

**Comment by voldemortX** [comment]
LGTM!

- [ ] Try use DDP
- [ ] Try SCNN style ERFNet
- [ ] Tune lr on validation set

**Comment by LittleJohnKhan** [comment]
erfnet_v2 with ERFNet as backbone

**Review by voldemortX — APPROVED** [review]
